Database/ActivitiesRegistry.sql

```diff
@@ -1,4 +1,4 @@
-use test;
+use ahmed_test;

```

Comment:
This is uncorrect, why are we using the database name from a simple file and changing the name eventually, this is not correct and should be fixed

Reply:
This is a whole new branch called MIOT-database-settings. It's better if we just move to another branch and let the server work well for everybody jejej.